language/transaction-builder-generator/Cargo.toml

```diff
@@ -10,10 +10,11 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1.0.31"
+heck = "0.3.1"
```

Comment:
Annoying question, but: is it possible for us to hand-implement the functionality we're using from this to avoid adding the dep?

Reply:
@sblackshear @metajack @bmwill `abigen` and `serde-generate` also need this. A minimal version would have to support at least snake case, camel case, and shouty snake case for plain ASCII characters. I suppose it could be developed within the `serde-reflection` repository (and perhaps even provided as codegen helper functions in `serde-generate`).